accounts-db/src/accounts_index/bucket_map_holder.rs

```diff
@@ -28,6 +30,17 @@ const _: () = assert!(std::mem::size_of::<Age>() == std::mem::size_of::<AtomicAg
 //   index is entirely disabled!  But there were concerns about the in-mem index growth behavior.
 // - 2 seconds is much faster, and does also reduce disk iops quite a lot.
 const AGE_MS: u64 = 2_000;
+// Trigger flushing when a bin exceeds this percent of the target entries
+const HIGH_WATER_MARK_PERCENTAGE: usize = 85;
+// Evict down to this percent of the target entries when flushing
+const LOW_WATER_MARK_PERCENTAGE: usize = 70;
+
+#[derive(Clone, Copy, Debug)]
+struct ThresholdEntriesPerBin {
+    target_entries: usize,
+    high_water_mark: usize,
+    low_water_mark: usize,
+}
```

Comment:
Please add documentation to the struct and fields.

Reply:
done.